src/components/csvLocalizer/jsonLocalizer.js

```diff
@@ -48,14 +48,32 @@ const JsonLocalizer = () => {
 
   const onCancel = React.useCallback(() => {
     setActiveCellId('');
-    setNewtranslation('');
+    setAlert('');
   }, []);
 
+  const activeCellContent =
+    activeCellId &&
+    data[activeCellId.split('-')[0]][+activeCellId.split('-')[1]];
+
   const saveTranslation = React.useCallback(() => {
-    console.log(`id: ${activeCellId}`, `translation: ${newTranslation}`);
+    if (activeCellContent === textareaRef.current.innerText) {
```

Comment:
Does the `textareaRef.current` always exist at this moment?

Reply:
Yes, at this moment it exists has innerText equal to content of active cell or it's own.